classes/passwd.oeclass

```diff
@@ -19,18 +19,14 @@ GROUP_FILES ?= "${SRCDIR}/group"
 PASSWD_POSTFUNC:machine = " do_install_passwd"
 do_install[postfuncs] += "${PASSWD_POSTFUNC}"
 do_install_passwd () {
-	i=0
 	for f in ${PASSWD_FILES} ; do
-		mkdir -p ${D}/${passwddir}
-		let i=$i+1
-		cp $f ${D}/${passwddir}/${PN}.$i
+		mkdir -p ${D}/${passwddir}/${PN}
+		cp $f ${D}/${passwddir}/${PN}/
```

Comment:
That's true. Could we make ${SRCDIR} implicit, so one should say PASSWD_FILES="foo/my-file bar/my-file", and then copy to the corresponding path under ${PN}? And instead of globbing we'd just do 'find ${passwddir} -type f | xargs cat >> ${passwdfile}'. Adding -n to cp is probably a good idea in any case.